src/devices/Lps22hb/README.md

```diff
@@ -0,0 +1,49 @@
+﻿# LPS22HB - Piezoresistive pressure and thermometer sensor
+
+Some of the applications mentioned by the datasheet:
+
+- Altimeter and barometer for portable devices
+- GPS applications
+- Weather station equipment
+- Sport watches
+
+## References
+
+- [LPS22HB datasheet](https://www.st.com/resource/en/datasheet/lps22hb.pdf)
+
+## How to use
+
+```csharp
+I2cConnectionSettings settings = new I2cConnectionSettings(1, Lps22hb.DefaultI2cAddress);
+
+I2cDevice device = I2cDevice.Create(settings);
+
+using Lps22hb sensor = new Lps22hb(device);
+
+while (true)
+{
+    var pressure = sensor.Pressure;
+    var temperature = sensor.Temperature;
+
+    Console.WriteLine($"Pressure: {pressure.Hectopascals:0.##}hPa");
+    Console.WriteLine($"Temperature: {temperature.DegreesCelsius:0.#}\u00B0C");
+
+    Thread.Sleep(1000);
+}
+```
+
+## Features bindings
+
+- [x] Interface I2c
+- [x] Read pressure (24 bit)
+- [x] Read temperature (16 bit)
+- [x] Continus mode (25Hz)
+- [x] BDU
+- [x] Enable Low-pass filter
```

Comment:
Looks a bit too good to me, when these items are checked but none of them are really available in the API as far as I can tell. What is BDU anyway?

Reply:
I added more available configuration.
_The BDU (Block Data Update) is used to inhibit the update of the output registers between the reading of upper, middle and lower register parts_
p20 => https://www.st.com/resource/en/application_note/an4833-measuring-pressure-data-from-sts-lps22hb-digital-pressure-sensor-stmicroelectronics.pdf